### PR TITLE
User Story 53: Admin User Index Page

### LINKED ISSUE #84: User Story 53, Admin User Index Page
As an admin user
When I click the "Users" link in the nav (only visible to admins)
Then my current URI route is "/admin/users"
Only admin users can reach this path.
I see all users in the system
Each user's name is a link to a show page for that user ("/admin/users/5")
Next to each user's name is the date they registered
Next to each user's name I see what type of user they are

### PR DESCRIPTION
**Pull request by mel-rob**
## Description
Resolves #84 
- Completed User Story 53: Admin User Index Page
- Added pirate-themed seeds 
- Added model tests for enable/disable methods (from User Stories 38-41)

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which maintains existing functionality)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## RSpec Results
```
Finished in 10.31 seconds (files took 2.16 seconds to load)
204 examples, 0 failures

Coverage report generated for RSpec to /Users/mel-rob/turing/mod_2/projects/monster_shop_part_1/coverage. 2522 / 2523 LOC (99.96%) covered.
```

### CODE REVIEW

File: app/views/admin/users/show.html.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (philjdelong) [inline app/views/admin/users/show.html.erb:1]: Ah yes, I like what you did here

File: spec/models/merchant_spec.rb [hunk 2]
```diff
@@ -66,5 +66,35 @@
...
+
+      expect(merchant_1.disabled).to eq(false)
+
+      expect(enabled_items.count).to eq(2)
+    end
+
+    it 'disable' do
+      merchant_1 = create(:merchant, name: 'Funny Pirate Name 1', disabled: false)
+
+      tire = merchant_1.items.create(name: "Gatorskins", description: "They'll never pop!", price: 100, image: "https://www.rei.com/media/4e1f5b05-27ef-4267-bb9a-14e35935f218?size=784x588", inventory: 12, disabled: false)
```
Comment (philjdelong) [inline spec/models/merchant_spec.rb:88]: Gatorskin tires

**Review by philjdelong — APPROVED** [review]
Looking good!

File: app/views/admin/users/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<h1>Admin User Dashboard</h1>
+
+<% @users.each do |user| %>
+  <section id="user-<%= user.id %>">
+    <%= link_to user.name, admin_user_path(user) %>
```
Comment (DanielEFrampton) [inline app/views/admin/users/index.html.erb:5]: Fancyyy

**Review by DanielEFrampton — APPROVED** [review]
Looks great!

**Pull request merged**